### ISSUE TITLE
Crashes when using schema.org JSON-LD

### ISSUE THREAD

**Issue by benmccann** [sveltejs/prettier-plugin-svelte#414]
The following `.svelte` file will cause a crash:
```
<svelte:head>
	<script type="application/ld+json">
	{
		"@context": "http://schema.org",
		"@type": "Corporation",
		"name": "C3 Ventures",
		"description": "C3 Ventures invests in and advises early stage ventures.",
		"naics": "523910",
		"url": "https://c3.ventures",
		"logo": "https://c3.ventures/images/c3-logo-white.png"
	}
	</script>
</svelte:head>
```

**Comment by curran** [comment]
Seems related to 

https://github.com/sveltejs/prettier-plugin-svelte/issues/419

**Comment by dummdidumm** [comment]
There should only be a log, but it's harmless, file formatting continues to work.
Either way, closed by #424
